keyboards/tada68/tada68.h

```diff
@@ -55,21 +54,23 @@
 // The first section contains all of the arguments
 // The second converts the arguments into a two-dimensional array
 
-#define LAYOUT_iso( \
+#define LAYOUT_65_iso( \
     k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, k0c, k0d, k0e, \
     k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c,      k1e, \
-    k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, k2c, k2d, k2e, \
+    k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, k1d, k2d, k2e, \
     k30, k31, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c, k3d, k3e, \
     k40, k41, k42,           k45,                k49, k4a, k4b, k4c, k4d, k4e  \
 ) \
 { \
     {k00, k01, k02, k03, k04, k05, k06, k07, k08, k09, k0a, k0b, k0c, k0d, k0e}, \
-    {k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, k2c, k1e}, \
+    {k10, k11, k12, k13, k14, k15, k16, k17, k18, k19, k1a, k1b, k1c, k1d, k1e}, \
     {k20, k21, k22, k23, k24, k25, k26, k27, k28, k29, k2a, k2b, XXX, k2d, k2e}, \
     {k30, k31, k32, k33, k34, k35, k36, k37, k38, k39, k3a, k3b, k3c, k3d, k3e}, \
     {k40, k41, k42, XXX, XXX, k45, XXX, XXX, XXX, k49, k4a, k4b, k4c, k4d, k4e}  \
 }
 
+#define LAYOUT_iso LAYOUT_65_iso
```

Comment:
Dont we also need backward compat for `LAYOUT_ansi` given it was also changed?
```suggestion
#define LAYOUT_iso LAYOUT_65_iso
#define LAYOUT_ansi LAYOUT_65_ansi
```

Reply:
It seems to have already been there, so it won't show up in the diff. It's on line 39 (40 master)

Reply:
:point_up_2: